app/conditions/ConditionsIndexLayout.tsx

```diff
@@ -12,8 +12,11 @@ const styles = {
   conditionsContainer: "flex flex-cols md:justify-around mx-auto",
   lettersContainer:
     "hidden md:block h-fit bg-[#F1F1F1] text-[24px] p-4 rounded mr-12",
-  letter: "flex flex-cols p-1 bold hover:text-[#0076AD] hover:underline",
-  termsContainer: "grid grid-col gap-5",
+  letter:
+    "flex flex-cols justify-center p-1 bold hover:text-[#0076AD] hover:underline",
+  greyedOutLetter: "text-gray-400 pointer-events-none",
+  termsContainer:
+    "flex-grow grid grid-cols-1 gap-5 overflow-auto max-h-[calc(73rem)] p-4",
```

Comment:
Curious where the 73 rem came from. I'm wondering if this may work differently on larger screens.